metaci/release/utils.py

```diff
@@ -55,38 +78,117 @@ def update_release_from_github(release, repo_api=None):
     return release
 
 
-def send_release_webhook(project_config, release):
+def send_release_webhook(project_config, release, config_item=None):
     if release is None or not settings.METACI_RELEASE_WEBHOOK_URL:
-        return
+        return  # should we better error handle this?
```

Comment:
This is not an error condition; we're intentionally keeping it possible to run an instance of MetaCI that isn't integrated with Change Traffic Control.